From d2e2a62ad09ae7b2fd2b906bf6a3dc7a53e6e14d Mon Sep 17 00:00:00 2001
From: Mark Walters <markwalters1009@gmail.com>
Date: Sat, 10 Sep 2016 18:07:26 +0100
Subject: [PATCH] Use dex-autostart to run xdg desktop files

We set dex-autostart to run all the desktop files for XFCE as there
are some things we do want (in particular xscreensaver) that are
listed as XFCE only. It may start some things that are not needed,
(eg xfsettingsd) but this does not seem to cause a problem.

Signed-off-by: Bahtiar `kalkin-` Gadimov <bahtiar@gadimov.de>
---
 awesomerc.lua | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/awesomerc.lua b/awesomerc.lua
index 0130b0d7..7f4c9954 100644
--- a/awesomerc.lua
+++ b/awesomerc.lua
@@ -582,3 +582,7 @@ end)
 client.connect_signal("focus", function(c) c.border_color = beautiful.border_focus end)
 client.connect_signal("unfocus", function(c) c.border_color = beautiful.border_normal end)
 -- }}}
+
+-- Use dex to run the xdg autostart files
+-- for the moment run the ones for XFCE
+awful.util.spawn_with_shell("dex-autostart -a -e XFCE")
-- 
2.14.1

